css/styles/navbar.css

```
@@ -0,0 +1,148 @@
+nav{
+    font-size: 90%;
+    display: flex;
+    width: calc(100% - 20px);
+    padding: 10px;
+    box-shadow: 0px 3px 3px rgb(195, 188, 188);
+    gap: 12px;
+    justify-content: center;
+    align-items: center;
+}
+
+.logo{
+    width: 90px;
+}
+
+.nav-items{
+    display: inline-block;
+}
+
+.link{
+    color: rgb(3, 3, 53);
+    text-decoration: none;
+    white-space: nowrap;
+}
+
+.link:visited{
+    color: rgb(3, 3, 53);
+}
+
+.link:hover{
+    color: rgb(100, 7, 100);
+}
+
+.search-bar{
+    width: 60%;
+    border: 1px solid rgb(15, 15, 15);
+    border-radius: 40px;
+    padding: 10px;
+    background-color: rgb(244, 244, 244);
+}
+
+.submit{
+    background-color: transparent;
+    border: none;
+    outline-width: 0;
+}
+
+.search-field{
+    border:none;
```

add a space after the colon.
```suggestion
    border: none;
```